cmd/ulxly/ulxly.go

```diff
@@ -1319,8 +1260,8 @@ func claimSingleDeposit(cmd *cobra.Command, client *ethclient.Client, bridgeCont
 			Uint32("OrigNet", deposit.OrigNet).
 			Uint32("DestNet", deposit.DestNet).
 			Uint32("NetworkID", deposit.NetworkID).
-			Str("OrigAddr", deposit.OrigAddr).
-			Str("DestAddr", deposit.DestAddr).
+			Str("OrigAddr", deposit.OrigAddr.String()).
+			Str("DestAddr", deposit.DestAddr.String()).
```

Comment:
`stringer` is worth considering here

```suggestion
			Stringer("OrigAddr", deposit.OrigAddr).
			Stringer("DestAddr", deposit.DestAddr).
```